spark/v3.4/spark/src/main/java/org/apache/iceberg/spark/actions/MigrateTableSparkAction.java

```diff
@@ -102,6 +102,14 @@ public MigrateTableSparkAction dropBackup() {
     return this;
   }
 
+  @Override
+  public MigrateTableSparkAction withBackupTableName(String tableName) {
+    if (!tableName.isEmpty()) {
```

Comment:
This behavior for checking if `tableName` is empty seems a bit weird to me. If this method is called, I assume someone wants to override the backup table name. I think we should just go ahead and set `backupIdent`.

I know destination and source identifiers are the same but it would be more readable to use `sourceTableIdent` in this case. We are backing up the source table, not the destination.

Reply:
Thanks for the suggestion.  Your suggestion is correct and I totally agree with it. Update as follows:
* remove checking if the table name is empty, and directly set the backup table name to `backupIdent`
* set the `sourceTableIdent` in the `backupIdent` update part.